llama_stack/templates/remote-vllm/run.yaml

```diff
@@ -122,7 +119,7 @@ models:
 - metadata:
     embedding_dimension: 384
   model_id: all-MiniLM-L6-v2
```

Comment:
need to pick a model from here? https://docs.vllm.ai/en/v0.7.0/models/supported_models.html#text-embedding-task-embed